lib/brakeman/checks/check_render.rb

```diff
@@ -94,4 +95,21 @@ def safe_param? exp
       end
     end
   end
-end 
+
+  def renderable? exp
+    call?(exp) and
+      sexp?(exp.target) and
+      constant?(exp.target) and
+      method_defined?(class_name(exp.target), :render_in)
+  end
+
+  def method_defined? class_name, name, seen = []
```

Comment:
I'm not sure about where best to put this, it feels too general too be scoped to only checking render calls. I also wanted to search `#includes` for `render_in` but the collection was always empty for my test components. I could see includes being populated for models but not for libs ... I could look better into how to populate includes? However, without that I _think_ this should address the ViewComponent use case as it looks like `render_in` is defined on `ViewComponent::Base`.